src/app/extensions/quoting/pages/quote-request-edit/quote-request-edit-page.component.ts

```diff
@@ -81,4 +92,21 @@ export class QuoteRequestEditPageComponent implements OnInit, OnDestroy {
   ngOnDestroy() {
     this.destroy$.next();
   }
+
+  private patchBreadcrumbData() {
```

Comment:
please do it via effects, see 1daca60b7e064ff2c469cc48513ed7c9ad953362 as an example